blog_project/blog_app/models.py

```diff
@@ -1,3 +1,16 @@
 from django.db import models
+from django.contrib.auth.models import User
+from datetime import date
+    
+class Blog(models.Model):
+    title=models.CharField(max_length=80)
```

Comment:
Set max length to more. Think about real life scenario. You have seen many blogs, have you faced any blog which have more than 80 characters? Those types of blog also exist. So set some reasonable limit.